### ISSUE TITLE
Add Cupertino widgets for iOS

### ISSUE THREAD

**Issue by rolandgeider** [wger-project/flutter#160]
While the app can be built for iOS without problems, it uses material design and looks out of place. This will be specially important when/if this is released on the app store.

Hopefully we can go a long way by simply using in a couple of places a Cupertino widget instead of the material ones.

https://docs.flutter.dev/development/ui/layout/building-adaptive-apps

**Comment by TaarnStar** [comment]
Hi, I would like to try to contribute to this issue

**Comment by rolandgeider** [comment]
Hi @TaarnStar ! Sure the issue is yours. Depending on what we want to do, this could be a huge ticket but I would suggest we exchange/add only the widgets necessary to make the app not feel complelety out of place

**Comment by TaarnStar** [comment]
Hi @rolandgeider, I found this package that we could implement for easier transitions between designs. Do you think I should try implement it or it would be too much work?

https://pub.dev/packages/flutter_platform_widgets

**Comment by rolandgeider** [comment]
Interesting. So instead of manually chacking we would just use their Platform Widgets. In any case this will be quite a bit of work, but we don't need to change  absolutely everything, just making the app not feel absolutely out of place would be a good start. Plus we shouldn't forget that we don't have any iOS releases yet (but we might do something for macos in the future)

**Comment by TaarnStar** [comment]
Maybe we can start by implementing the package to use with Icons (it is maybe the easiest thing to change), either that or manually checking for the platform?

**Comment by rolandgeider** [comment]
Yeah. We can try the package, then we keep the code cleaner and don't have to litter everything with ifs

**Comment by TaarnStar** [comment]
I just found which icons are available through the package so we will have to use on some icons the ifs (if there is a cupertino version of some of those icons), but for a lot of icons we can use the package

https://github.com/stryder-dev/flutter_platform_widgets/blob/master/ICONS.md

**Comment by rolandgeider** [comment]
Nice. I think we only use the only "weirder" Icon is the ine for the weight scale, but that's ok if it stays
